In [23]:
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split

In [24]:
def load_dataset(filename):
    data = pd.read_csv(filename, header=None)
    dataset = data.values
    #split to input x and output y
    x = dataset[:,:-1]
    y = dataset[:,-1]

    x = x.astype(str) #format values as string
    y = y.reshape((len(y),1)) #reshape target to 2d array

    return x, y
    

In [26]:
x, y = load_dataset('breast-cancer.csv')
#split intro train and test sets
x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=0.33, random_state=1)

In [27]:
print('Train', x_train.shape, y_train.shape)
print('Test', x_test.shape, y_test.shape)

Train (191, 9) (191, 1)
Test (95, 9) (95, 1)


We have 191 samples for training and 95 for testing.